servers/admin_ui/common.py

```diff
@@ -0,0 +1,82 @@
+# Define an Enum for secret provider options
+import os
+from enum import Enum
+
+import streamlit as st
+
+from agent_guard_core.credentials.aws_secrets_manager_provider import AWSSecretsProvider
+from agent_guard_core.credentials.conjur_secrets_provider import ConjurSecretsProvider
+from agent_guard_core.credentials.environment_manager import EnvironmentVariablesManager
+from agent_guard_core.credentials.file_secrets_provider import FileSecretsProvider
+
+# Define constants for configuration keys
+CONJUR_AUTHN_LOGIN_KEY = "CONJUR_AUTHN_LOGIN"
+CONJUR_AUTHN_API_KEY_KEY = "CONJUR_AUTHN_API_KEY"
+CONJUR_APPLIANCE_URL_KEY = "CONJUR_APPLIANCE_URL"
+SECRET_PROVIDER_KEY = "SECRET_PROVIDER"
+
+
+class SecretProviderOptions(Enum):
+    AWS_SECRET_MANAGER = "AWS Secret Manager"
```

Comment:
In AWS you called it secret manager while the other two are secret provider. Maybe we should align the standard?
Also, AWS Secrets Manager is names with a plural "secrets"